mdm/api/src/main/java/com/gentics/mesh/core/data/dao/NodeDao.java

```diff
@@ -364,4 +385,161 @@ default HibNode create(HibNode parentNode, HibUser creator, HibSchemaVersion sch
 	 * @param branchUuid
 	 */
 	void setPublished(HibNode node, InternalActionContext ac, HibNodeFieldContainer container, String branchUuid);
+
+	/**
+	 * Stream the hierarchical patch of the node.
+	 * 
+	 * @param node
+	 * @param ac
+	 * @return
+	 */
+	Stream<HibNode> getBreadcrumbNodeStream(HibNode node, InternalActionContext ac);
```

Comment:
Here I've added all the methods that are implemented in `PersistingNodeDao`, since I saw this was the status quo for other daos implementation. However, now that we have `CommonTx` that gives us access to the `PersistingDao`, I think we can get rid of them, right? @plyhun

Reply:
See no problems with the current impl, so leaving it.